halo/comet/comet.go

```diff
@@ -64,6 +68,9 @@ func (a adapter) IsValidator(ctx context.Context, valAddress common.Address) boo
 // Validators returns the cometBFT validators at the given height or false if not
 // available (probably due to snapshot sync after height).
 func (a adapter) Validators(ctx context.Context, height int64) (*cmttypes.ValidatorSet, bool, error) {
+	ctx, span := tracer.Start(ctx, "comet/validators")
```

Comment:
you can add attributes to the span, for example, the height parameter in this case.

Reply:
yeah sure we should add this